http/src/main/resources/reference.conf

```diff
@@ -224,7 +224,7 @@ gke {
     # only need to be unique within a namespace, but something in the Galaxy chart requires them
     # to be unique within a cluster.
     releaseNameSuffix = "gxy-rls"
-    chartName = "galaxy/galaxykubeman"
+    chartName = "/leonardo/galaxykubeman"
```

Comment:
why are the `galaxy` to `leonardo` changes?

Reply:
Previously the charts were downloaded from the remote `galaxy` repo (which points to GitHub).

Now the charts are cached on disk in the container in the `/leonardo` directory.

Reply:
There directory is specified explicitly on line 43 of the dockerfile ` pushd /leonardo && \`

Reply:
I see 👍🏼